tests/console/login.go

```diff
@@ -139,10 +139,15 @@ func LoginToFedora(vmi *v1.VirtualMachineInstance) error {
 		&expect.BSnd{S: "sudo su\n"},
 		&expect.BExp{R: PromptExpression},
 	})
-	res, err := expecter.ExpectBatch(b, 3*time.Minute)
-	if err != nil {
-		log.DefaultLogger().Object(vmi).Infof("Login: %+v", res)
-		return err
+	res, err := expecter.ExpectBatch(b, 2*time.Minute)
+	if err != nil {
+		log.DefaultLogger().Object(vmi).Reason(err).Errorf("Login: %+v", res)
+		// Try once more since sometimes the login prompt is ripped apart by asynchronous daemon updates
+		res, err := expecter.ExpectBatch(b, 1*time.Minute)
+		if err != nil {
+			log.DefaultLogger().Object(vmi).Reason(err).Errorf("Login: %+v", res)
```

Comment:
Can you state in the log that this is a retry ?

Reply:
Done.